src/frontend/site.js

```diff
@@ -328,16 +341,65 @@ function postBio(req, res) {
         .then((_) => {
             res.redirect(extendURL(`/user/${getJwtUser(req.cookies)}`));
         }).catch(error => {
-            res.status(statusCodeForError(error)).render('error.njk', handleError(error));
-        });
+        res.status(statusCodeForError(error)).render('error.njk', handleError(error));
+    });
 }
 
-/**
+function postLike(req, res) {
```

Comment:
[NEED_INFO] could we also make this function async? Then we can greatly decrease the nesting level here and improve readability.

Reply:
Function made Async and nesting level reduced.